identity_credential/src/revocation/document_ext.rs

```diff
@@ -30,7 +31,6 @@ pub trait RevocationDocumentExt: private::Sealed {
   ///
   /// Fails if the referenced service is not found, or is not a
   /// valid `RevocationBitmap2022` service.
-  #[cfg(feature = "revocation-bitmap")]
   fn resolve_revocation_bitmap(&self, query: DIDUrlQuery<'_>) -> RevocationResult<RevocationBitmap>;
```

Comment:
I wonder about the reason behind this.

Reply:
It's unnecessary since the entire module is already `[cfg(feature = "revocation-bitmap")]`.